src/transforms/glossary.ts

```diff
@@ -27,11 +55,35 @@ export function addGlossaryDefinition(
 
         if (at == null) return
 
+        const fragment: Descendant[] = Editor.fragment(editor, at)
+        const nodeToInsert: Node[] = []
+        const getToBottom = (descendant: Descendant[]) => {
+            if (Array.isArray(descendant)) {
```

Comment:
`Editor.fragment` returns `Descendant[]`, so this check is unnecessary; it is always an array.

Reply:
About this check - 
I also thought that extra checking was unnecessary, but if it's not there then I have a bug:
`TypeError: Cannot read properties of undefined`